### PR TITLE
perf report for 2023-09-13

### PR DESCRIPTION
**Pull request by pnkfelix**

### CODE REVIEW

File: draft/2023-09-13-this-week-in-rust.md [hunk 1]
```diff
@@ -71,7 +71,22 @@ If you are a Rust project owner and are looking for contributors, please submit
...
+Beyond that, we had a scattering of small regressions to instruction-counts that
+were justified because they were associated with bug fixes.
+
+Triage done by **@pnkfelix**.
+Revision range: [15e52b05..7e0261e7](https://perf.rust-lang.org/?start=15e52b05ca8f63e0da27c808680388717e5b997e&end=7e0261e7ea2085bdc0bc3d0fd6776bf343473858&absolute=false&stat=instructions%3Au)
+
+3 Regressions, 2 Improvements, 5 Mixed; 2 of them in rollups
+84 artifact comparisons made in total
+
+[Full report here](https://github.com/rust-lang/rustc-perf/blob/master/triage/2023-09-13.md)
```
Comment (mariannegoldin) [inline draft/2023-09-13-this-week-in-rust.md:89]: This link seems to be broken - is there another target we should point to?

**Review by mariannegoldin — COMMENTED** [review]
Thank you for this submission. As soon as we can update the full report link target, I will approve and merge.

**Comment by pnkfelix** [comment]
> Thank you for this submission. As soon as we can update the full report link target, I will approve and merge.

The full report will be available once https://github.com/rust-lang/rustc-perf/pull/1719 is merged. I’ll go check if there’s someone who can approve it (though I think there may be precedent to allow this notice to be merged to TWIR before the rustc-perf PR is merged…)

**Comment by mariannegoldin** [comment]
@pnkfelix thanks for the update and the PR link! We might post the summary, and then once the PR goes through, add the link to the full report.

File: draft/2023-09-13-this-week-in-rust.md (hunk 1 above)
Comment (mariannegoldin) [inline draft/2023-09-13-this-week-in-rust.md:89]:
```suggestion
[Full report  - pending](https://github.com/rust-lang/rustc-perf/blob/master/triage/2023-09-13.md)
```

**Review by mariannegoldin — APPROVED** [review]
Approving and merging with "pending" annotation for link - to be updated once the report is published.

**Pull request merged**